src/main/webapp/app/entities/category/category.route.ts

```diff
@@ -73,6 +78,51 @@ export const categoryRoute: Routes = [
       pageTitle: 'learnportalApp.category.home.title'
     },
     canActivate: [UserRouteAccessService]
+  },
+  {
+    path: ':categoryId/theory',
```

Comment:
Change all `categoryId` to simple `id` as we use it in category route

Reply:
Done.